gratipay/models/package/__init__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from postgres.orm import Model
+
+
+class Package(Model):
+    """Represent a gratipackage. :-)
+    """
+
+    typname = 'packages'
+
+    def __eq__(self, other):
+        if not isinstance(other, Package):
+            return False
+        return self.id == other.id
+
+    def __ne__(self, other):
+        if not isinstance(other, Package):
+            return True
+        return self.id != other.id
+
+
+    # Constructors
+    # ============
+
+    @classmethod
+    def from_id(cls, id):
+        """Return an existing package based on id.
+        """
+        return cls.db.one("SELECT packages.*::packages FROM packages WHERE id=%s", (id,))
+
+    @classmethod
+    def from_names(cls, package_manager, name):
+        """Return an existing package based on package manager and package names.
+        """
+        return cls.db.one("SELECT packages.*::packages FROM packages "
+                          "WHERE package_manager=%s and name=%s", (package_manager, name))
```

Comment:
> Is there plan to create table for package manager and reference its id in packages table?

Not before May. We won't look at expanding beyond npm until later this year or possibly next.

> Before making that change, shall we make 'npm' a constant shared across the app to avoid issue with raw string? Like using "Npm" by mistake, etc.

Like 9c052766cd54a80c3885808d0c2e5e0aeb9afd1f?